Mathlib/Algebra/QuadraticDiscriminant.lean

```diff
@@ -98,6 +98,21 @@ theorem quadratic_eq_zero_iff_of_discrim_eq_zero (ha : a ≠ 0) (h : discrim a b
   have : discrim a b c = 0 * 0 := by rw [h, mul_zero]
   rw [quadratic_eq_zero_iff ha this, add_zero, sub_zero, or_self_iff]
 
+theorem discrim_eq_zero_of_existsUnique (ha : a ≠ 0) :
```

Comment:
Is there a reason why you have the `→` in the statement? We usually write it as an explicit assumption.

Reply:
Would you prefer I start with `revert` instead?

Reply:
No preference.